roles/config-gnome/tasks/gnome.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Install additional packages for Gnome"
```

Comment:
yeah, thinking it may be supported for Fedora only at this point ... either way I think the change is good, but only do it for Fedora.